pkg/action/server.go

```diff
@@ -280,6 +289,40 @@ func handler(cfg *config.Config, db store.Store, logger *slog.Logger, client *gi
 						w.Header().Set("Content-Type", "text/plain")
 						w.WriteHeader(http.StatusInternalServerError)
 
+						io.WriteString(w, http.StatusText(http.StatusInternalServerError))
+					}
+				case *github.WorkflowJobEvent:
+					wf_job := event.WorkflowJob
```

Comment:
Please use the getters for this and the following lines. The vet check is already failing below because of the types.